lib/dealership.rb

```diff
@@ -0,0 +1,47 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory_count = @inventory.length
+  end
+
+  def has_inventory?
+    if @inventory_count < 2
+      return false
+    end
+  end
+
+  def cars_by_make(make_check)
+    @cars_by_make = []
+
+    @inventory.each do |car|
+      if car.make == make_check
+        @cars_by_make << car
+      end
+    end
+    return @cars_by_make
+  end
+
+  def total_value
+    value=0
+
+    @inventory.each do |car|
```

Comment:
Nice job using the `.each` here with your counter. Look into the `.sum` enumerable. Since what we are wanting to do is return the sum of our array.
https://ruby-doc.org/core-2.4.1/Enumerable.html#:~:text=sum-,(,-init%3D0)%20%E2%86%92%20number